### PR TITLE
Complete french translation for Confirm Rename view

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

This changes can be skipped for the changelog ^^.

### Desired reviewers

 @dduportal or @smerle33 or @lemeurherve or @gounthar or @olamy

### CODE REVIEW

**Review by gounthar — APPROVED** [review]

**Review by lemeurherve — APPROVED** [review]
Thanks!

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
